roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -125,13 +125,15 @@
 - name: Generate proxy session
   set_fact:
     session_secret: "{{ 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789' | random_word(200) }}"
+  when: session_secret is not defined
```

Comment:
I had been struggling with trying to figure out how we should handle this. What if we install a non-ops and then go back and install with ops enabled? we should reuse the prior session secret but the fact may not exist. Similar for the oauth secret.

Should we write these values to a file if they don't exist and then read from them similar to how we do for our certs?